.gitlab-ci.yml

```diff
@@ -544,13 +543,12 @@ publish:s3:mender-monitor:automatic:
     # Make copies in known destinations to be consumed by get.mender.io script:
     # * For "experimental" channel, we make a copy of the newest master package in the master directory
     # * For "stable" channel, we make a copy in a separate "latest" subdirectory of the latest tagged version
-
-    # Needs to be reworked, see MEN-5029
-    # - if is_build_tag ${MENDER_GATEWAY_VERSION} || [ "${MENDER_GATEWAY_VERSION}" == "master" ]; then
-    # -   for arch in amd64 arm64 armhf; do
-    # -     aws s3 cp output/commercial/mender-gateway*_${arch}.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/master/mender-gateway_master-1_${arch}.deb
-    # -  done
-    # - fi
+    - if [ "${MENDER_GATEWAY_VERSION}" == "master" ]; then
+    -   for arch in amd64 arm64 armhf; do
+    -     aws s3 cp output/commercial/mender-gateway*_${arch}.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/master/mender-gateway_master-1+debian+buster_${arch}.deb
```

Comment:
aws s3 cp does not accept many files -- I broke the pipelines with that last time. it will only work when there is a single file matching `output/commercial/mender-gateway*_${arch}.deb`

Reply:
okey, I added the comment there, so we do not forget.